resourcew.h

```diff
@@ -47,7 +48,7 @@
 #ifndef APSTUDIO_READONLY_SYMBOLS
 #define _APS_NEXT_RESOURCE_VALUE        159
 #define _APS_NEXT_COMMAND_VALUE         40001
-#define _APS_NEXT_CONTROL_VALUE         1020
+#define _APS_NEXT_CONTROL_VALUE         1021

```

Comment:
The Win32 API continues to entertain me with its naivety.

Reply:
Speaking of this, I feel the need to mention that there's are `IDC_CHECK1` and `IDC_COMBO1` defines that have the same value as something else and are completely unused.

Reply:
We should probably get rid of those, but they're most likely just artifacts from visual studio and aren't causing any trouble.